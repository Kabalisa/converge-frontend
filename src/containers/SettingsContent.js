import React from 'react';
import SettingsNav from '../components/SettingsNav';
import OfficeList from '../components/OfficeList';
import '../assets/styles/settingscontent.scss';

class SettingsContent extends React.Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return (
      <div className="settings-vertical">
        <SettingsNav />
        <OfficeList />
      </div>
    );
  }
}

export default SettingsContent;
