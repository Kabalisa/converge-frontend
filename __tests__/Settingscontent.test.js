import React from 'react';
import { shallow } from 'enzyme';

import SettingsContent from '../src/containers/SettingsContent';

describe('Tests for SettingsContent', () => {
  const shalloWrapper = shallow(<SettingsContent />);

  it('renders correctly in memory', () => {
    expect(shalloWrapper).toMatchSnapshot();
  });
});
