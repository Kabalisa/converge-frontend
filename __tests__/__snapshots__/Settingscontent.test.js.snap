// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for SettingsContent renders correctly in memory 1`] = `
<div
  className="settings-vertical"
>
  <SettingsNav />
  <OfficeList />
</div>
`;
